backend/database/migrations/2022_11_16_134040_create_questions_table.php

```diff
@@ -15,13 +15,14 @@ public function up()
     {
         Schema::create('questions', function (Blueprint $table) {
             $table->uuid('id')->unique();
-            $table->string("question");
             $table->string("options");
             $table->string("timeframe");
             $table->string("correct_answers");
+            $table->string("question");
             $table->boolean("is_multiple_answers");
```

Comment:
this is not accepted.. pls use the update migration method instead..

Reply:
Why did you touch the migration file in the first place?

Reply:
i'll remove it, i think it the update migration for this already exists

Reply:
you didn't return the question_id column..

Reply:
it's no more in the dev branch

Reply:
ok